pkg/istio/rbac/common/servicerolebinding_test.go

```diff
@@ -136,17 +138,21 @@ func TestGetServiceRoleBindingSpec(t *testing.T) {
 		{
 			test: "empty args",
 			input: input{
+				athenzDomainName:       "",
+				roleName:               "",
 				k8sRoleName:            "",
 				members:                nil,
 				enableOriginJwtSubject: true,
 			},
 			expectedSpec: nil,
-			expectedErr:  fmt.Errorf("no subjects found for the ServiceRoleBinding: "),
+			expectedErr:  fmt.Errorf("empty string found in athenzDomainName:  and roleName: "),
 		},
 		{
 			test: "valid role member spec",
 			input: input{
-				k8sRoleName: "client-reader--role",
+				athenzDomainName: "athenz.domain",
+				roleName:         "rolename",
```

Comment:
since `k8sRoleName` is derived from `roleName`, `roleName` should be `client-reader_role`

Reply:
the unit tests are failing because of this, please update the other tests as well